<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>端口设置</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>端口   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="port_comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudrate_comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>数据读取间隔（s)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="read_interval_doubleSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="auto_detect_pushButton">
           <property name="text">
            <string>自动检测</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connect_pushButton">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>数据读取</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="read_once_pushButton">
           <property name="text">
            <string>单次读取</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="read_serious_pushButton">
           <property name="text">
            <string>开始连续读取</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_pushButton">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chart_tab">
       <attribute name="title">
        <string>图表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="data_tab">
       <attribute name="title">
        <string>数据</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>保存为...</string>
     </property>
     <addaction name="actionCSV"/>
     <addaction name="actionExcel"/>
     <addaction name="actionAccess"/>
    </widget>
    <addaction name="action"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
  <action name="actionAccess">
   <property name="text">
    <string>Access</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="install_driver_action">
   <property name="text">
    <string>安装USB驱动</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
